<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lbl_yunshu">
          <property name="text">
           <string>韵书</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbbox_yushu">
          <item>
           <property name="text">
            <string>平水韵</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>词林正韵</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>中华新韵</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_gelv">
          <property name="text">
           <string>格律</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbbox_gelv">
          <item>
           <property name="text">
            <string>五言</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>七言</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>六言</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_ticai">
          <property name="text">
           <string>体裁</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbbox_ticai">
          <item>
           <property name="text">
            <string>律诗</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>绝句</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>排律</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
        <item>
         <widget class="QPlainTextEdit" name="te_input">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ErrorWidget" name="widget" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_check">
          <property name="text">
           <string>查重</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_range">
          <property name="text">
           <string>范围</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tbl_dup">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ErrorWidget</class>
   <extends>QWidget</extends>
   <header location="global">errorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
